arelle/plugin/validate/EDINET/rules/gfm.py

```diff
@@ -813,3 +783,106 @@ def rule_gfm_1_3_31(
             msg=_("Set the abstract attribute to 'true'."),
             modelObject=nonAbstractDomainElements
         )
+
+
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[DISCLOSURE_SYSTEM_EDINET],
+)
+def rule_gfm_1_5_6(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    EDINET.EC5700W: [GFM 1.5.6] The length of a label must be less than 511 characters unless it its role is documentation.
```

Comment:
This description suggests the code should say `>= 511`, not `> 511`.

nit: "it its role".

Reply:
GFM.1.5.6: `The text of link:label must be a string of fewer than 511 characters unless its
xlink:role attribute is ‘http://www.xbrl.org/2003/label/documentation’.`

So it must be ` > 511`.